app/renderer/components/tabs/tabPage.js

```diff
@@ -144,19 +147,54 @@ class TabPage extends React.Component {
     return <span
       ref={(node) => { this.tabPageNode = node }}
       data-tab-page={this.props.index}
+      data-test-id='tabPage'
+      data-test2-id={this.props.index}
+      data-test-active-tabPage={this.props.active}
       onDragOver={this.onDragOver.bind(this)}
       onMouseEnter={this.props.previewTabPage ? this.onMouseEnter : null}
       onMouseLeave={this.props.previewTabPage ? this.onMouseLeave : null}
       onDrop={this.onDrop.bind(this)}
-      className={cx({
-        tabPage: true,
-        audioPlaybackActive: this.props.isAudioPlaybackActive,
-        active: this.props.active
-      })}
+      className={css(
+        styles.tabPage,
+        this.props.isAudioPlaybackActive && styles.tabPage_audio,
+        this.props.active && styles.tabPage_active
+      )}
       onContextMenu={this.onContextMenu}
       onClick={this.onClick}
     />
   }
 }
 
+const styles = StyleSheet.create({
+  tabPage: {
+    backgroundColor: theme.tabPage.backgroundColor,
+    border: `1px solid ${theme.tabPage.borderColor}`,
+    borderRadius: globalStyles.radius.borderRadius,
+    boxSizing: 'border-box',
+    display: 'inline-block',
+    margin: 'auto 2.5px',
+    height: '7px',
+    width: '35px',
+    WebkitAppRegion: 'no-drag',
+
+    ':hover': {
+      backgroundColor: theme.tabPage.backgroundColor,
+      border: `1px solid ${theme.tabPage.hover.borderColor}`
+    }
+  },
+
+  tabPage_audio: {
+    border: `1px solid ${theme.audio.color}`
+  },
+
+  tabPage_active: {
+    backgroundColor: theme.tabPage.active.backgroundColor,
+    border: 'none',
+
+    ':hover': {
+      border: `1px solid ${theme.tabPage.active.hover.borderColor}`
```

Comment:
active tab page should have the same state when hovering (filled orange)

Reply:
On theme.js, they are set like this:

````
    tabPage: {
      backgroundColor: '#fff',
      borderColor: '#bbb',

      hover: {
        borderColor: globalStyles.color.braveOrange
      },

      active: {
        backgroundColor: globalStyles.color.braveOrange,

        hover: {
          borderColor: globalStyles.color.braveOrange
        }
      }
````

So they should be same by default. Is it OK?

Reply:
not sure what is happening but this is what I'm seeing now:

![nov-03-2017 23-53-46](https://user-images.githubusercontent.com/4672033/32401239-59df0624-c0f2-11e7-9b27-a31ea1bdc432.gif)

Reply:
maybe you should enforce the background color on hover for the active tab page too

Reply:
Wow nice find! I addressed that here: https://github.com/brave/browser-laptop/pull/11530/commits/0582859ef29a0c5c0589b972fff17a31545543cf#diff-b39b5cd9b6e26e313c7387108e17891cR198